pubsub/pullstream_test.go

```diff
@@ -67,7 +67,7 @@ func TestPullStreamGet(t *testing.T) {
 			test.errors = test.errors[1:]
 			return &testStreamingPullClient{sendError: err}, nil
 		}
-		ps := newPullStream(context.Background(), streamingPull, "")
+		ps := newPullStream(context.Background(), streamingPull, "", 100, 100)
```

Comment:
```suggestion
		ps := newPullStream(context.Background(), streamingPull, "", 100, 1000)
```
nit: I prefer if the test used the default MaxOutstandingBytes value of 1000